### PR TITLE
Create partition with just collection name

### PR DESCRIPTION
**Pull request by im-ajaymeena**
Solves #878

### CODE REVIEW

File: pymilvus/orm/partition.py [hunk 1]
```diff
@@ -29,9 +28,13 @@
...
         from .collection import Collection
-        if not isinstance(collection, Collection):
-            raise CollectionNotExistException(message=ExceptionsMessage.CollectionType)
-        self._collection = collection
+        if isinstance(collection, Collection):
+            self._collection = collection
+        elif isinstance(collection, str):
+            self._collection = Collection(collection)
+        else:
+            raise ValueError("Collection must be od type pymilvus.Collection or a string.")
```
Comment (XuanYang-cn) [inline pymilvus/orm/partition.py:36]: Great optimize!

Please use MilvusException here

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: pymilvus/orm/partition.py (hunk 1 above)
Comment (im-ajaymeena) [reply pymilvus/orm/partition.py]: @XuanYang-cn 
Should we check for the case of empty string ""

Comment (XuanYang-cn) [reply pymilvus/orm/partition.py]: Collection could take care of the checks

**Comment by XuanYang-cn** [comment]
The actions's error has been fixed. Add a manual ci-pass
/lgtm
/approve

**Pull request merged**